osu.Game.Tournament/Screens/Ladder/Components/DrawableTournamentMatch.cs

```diff
@@ -344,6 +345,9 @@ public void Remove()
             Match.Progression.Value = null;
             Match.LosersProgression.Value = null;
 
+            if (ladderInfo == null)
+                return;
```

Comment:
why early return and not assert?

Reply:
This confuses me because I'm not sure if it's will null here, if test delete in test but here is not return, it will cause null exception